### ISSUE TITLE
Differences between matlab script and previous fsl shell script?

### ISSUE THREAD

**Issue by ccraddock** [asoroosh/DVARS#3]
The former fsl script dvars.sh and DVARSCalc.m give different values for RDVARS although perfectly correlated. Is this a problem? Which implementation is preferred?

**Comment by asoroosh** [comment]
Hi Cameron, 

*tl;dr:* to have a standardised DVARS measure which is agnostic to intensity normalisation and comparable across cohorts/subject, I would use `pDvar` from DSEvars.m. Note there are similar standardised measure for the slow components too. 

I suppose the implementation for Standardised (now called Relative) DVARS uses AR1 estimation available via `fslmaths` as suggested in the [original standardised DVARS work](https://arxiv.org/pdf/1704.01469) while `DVARSCalc.m` uses a robust estimate of autocorrelation via intraclass correlation coefficient -- see `madicc` function inside DVARSCalc. I have never seen dvars.sh so I can't certainly comment about its function.   
TBH, I prefer neither. To get a standardised DAVRS (which is comparable across cohorts/subjects and is invariant of intensity normalisation) you can use `pDvar` which is DVARS normalised by the whole variability (i.e. A-var) and is available via DSEvars.m. Alternatively you can also use `SDVARS_*` in DVARSCalc.m which is the back-projection of p-values to either a normal or Chi^2 distribution.
